@xen-orchestra/web/src/pages/settings.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <UiHeadBar icon="fa:gear">{{ t('settings') }}</UiHeadBar>
+  <UiCard class="settings">
+    <!-- ABOUT -->
+    <div class="container">
+      <UiTitle>{{ t('about') }}</UiTitle>
+      <VtsColumns class="columns">
+        <VtsQuickInfoColumn>
+        <!-- TODO Update all h6 title when component is updated -->
+          <div class="typo-h6">{{ t('xen-orchestra') }}</div>
+          <VtsQuickInfoRow :label="t('version')">
+            <template #value />
+          </VtsQuickInfoRow>
```

Comment:
Template value is not necessary
```suggestion
          <VtsQuickInfoRow :label="t('version')" />
```